unabridged-works-volume-4/original/01-further-testimony-to-truth.adoc

```diff
@@ -1157,17 +1218,17 @@ Where the light is arisen, there it is day;
 where the darkness covereth and possesseth the minds of men, there it is night.
 
 [.discourse-part]
-Question.
-5 How may a man know whether it be night or day with him?
+_Question 5._ How may a man know whether it be night or day with him?
 
 [.discourse-part]
 Answer.
 By waiting to feel somewhat of the life of God arising in him,
 by turning and hearkening to his witness.
 That will faithfully discover how it is with him, and where he is.
 
-Question 6. How may a man come out of the darkness of the night,
-into the light and brightness of the day?
+[.discourse-part]
+__Question 6. How may a man come out of the darkness of the night,
```

Comment:
The italics don't look exactly right to me here. Did you mean to have it be:

```ASCIIDOC
_Question 6._ How may a man come out of the darkness of the night,
```

Reply:
Yes, idk why that happened!